enhancements/api-review/gcp_user_defined_labels.md

```diff
@@ -0,0 +1,321 @@
+---
+title: gcp_user_defined_labels
+authors:
+  - "@bhb"
+reviewers:
+  - "@patrickdillon" ## reviewer for installer component
+  - "@JoelSpeed" ## reviewer for api and machine-api-provider-gcp components
+  - "@dmage" ## reviewer for cluster-image-registry-operator component
+  - "@akhil-rane" ## reviewer for cloud-credential-operator component
+approvers:
+  - "@jerpeter1" ## approver for CFE
+api-approvers:
+  - "@JoelSpeed" ## approver for api component
+creation-date: 2022-07-12
+last-updated: 2022-07-12
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-8155
+  - https://issues.redhat.com/browse/CORS-2249
+see-also:
+  - "enhancements/api-review/custom-tags-aws.md"
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+
+# Apply user defined labels to all GCP resources created by OpenShift
+
+## Summary
+
+This enhancement describes the proposal to allow an administrator of OpenShift to 
+have the ability to apply user defined labels to those resources created by OpenShift 
+in GCP.
+
+## Motivation
+
+Motivations include but are not limited to:
+
+- Allow admin, compliance, and security teams to keep track of assets and objects 
+  created by OpenShift in GCP.
+
+### User Stories
+
+- As an openshift administrator, I want to have labels added to all resources created 
```

Comment:
This user story is missing the why, the typical format is `As a <persona>, I want to <do something>, so that <I can achieve some goal>`, but we don't have the goal part included here, please add the goal to explain what the administrator will be able to achieve with the new feature

Reply:
comment incorporated.